app/src/main/java/com/example/myapplication/activity/MainActivity.kt

```diff
@@ -0,0 +1,70 @@
+package com.example.myapplication.activity
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.view.View
+import android.widget.Button
+import com.example.myapplication.R
+import com.example.myapplication.mvp.model.CalculatorModel
+import com.example.myapplication.mvp.presenter.CalculatorPresenter
+import com.example.myapplication.mvp.view.CalculatorView
+import kotlinx.android.synthetic.main.activity_main.button_divide
+import kotlinx.android.synthetic.main.activity_main.button_eight
+import kotlinx.android.synthetic.main.activity_main.button_equal
+import kotlinx.android.synthetic.main.activity_main.button_five
+import kotlinx.android.synthetic.main.activity_main.button_four
+import kotlinx.android.synthetic.main.activity_main.button_less
+import kotlinx.android.synthetic.main.activity_main.button_multiplication
+import kotlinx.android.synthetic.main.activity_main.button_nine
+import kotlinx.android.synthetic.main.activity_main.button_one
+import kotlinx.android.synthetic.main.activity_main.button_plus
+import kotlinx.android.synthetic.main.activity_main.button_point
+import kotlinx.android.synthetic.main.activity_main.button_seven
+import kotlinx.android.synthetic.main.activity_main.button_six
+import kotlinx.android.synthetic.main.activity_main.button_three
+import kotlinx.android.synthetic.main.activity_main.button_two
+import kotlinx.android.synthetic.main.activity_main.button_zero
+
+class MainActivity : AppCompatActivity(), View.OnClickListener {
+
+    private var presenter : CalculatorPresenter ?= null
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_main)
+        presenter = CalculatorPresenter(CalculatorModel(), CalculatorView(this))
+
+        button_zero.setOnClickListener(this)
+        button_one.setOnClickListener(this)
+        button_two.setOnClickListener(this)
+        button_three.setOnClickListener(this)
+        button_four.setOnClickListener(this)
+        button_five.setOnClickListener(this)
+        button_six.setOnClickListener(this)
+        button_seven.setOnClickListener(this)
+        button_eight.setOnClickListener(this)
+        button_nine.setOnClickListener(this)
+        button_plus.setOnClickListener(this)
+        button_equal.setOnClickListener(this)
+        button_less.setOnClickListener(this)
+        button_point.setOnClickListener(this)
+        button_divide.setOnClickListener(this)
+        button_multiplication.setOnClickListener(this)
```

Comment:
this code could be in a method, and be called on onCreate.
try that onCreate() keeps clean and simply

Reply:
done